package.json

```diff
@@ -61,5 +52,21 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@babel/core": "^7.16.7",
```

Comment:
Moved ones that are recommended to install in `devDependencies` to `devDependencies`